liblangutil/CharStream.cpp

```diff
@@ -73,6 +73,13 @@ char CharStream::rollback(size_t _amount)
 	return get();
 }
 
+char CharStream::setPosition(size_t _location)
+{
+	solAssert(_location <= m_source.size(), "Attemping to set position past end of source.");
```

Comment:
Typo `Attemping`

Reply:
Thanks! Looks like this has been addressed in commit af86f4274d